### ISSUE TITLE
Accessibility Error: ARIA attributes must conform to valid values

### ISSUE THREAD

**Issue by beckej13820** [omeka-s-modules/ItemCarouselBlock#11]
I have been doing automated accessibility testing on an Omeka S site using the Item Carousel Block. The accessibility tool I am using for the testing is Axe Dev Tools, a free browser extension created by Deque. https://www.deque.com/axe/devtools/

I am getting the error message "ARIA attributes must conform to valid values"
![Screen Shot 2022-05-14 at 10 25 47 AM](https://user-images.githubusercontent.com/29982981/168429935-5da23cad-f095-417b-b128-c0763ec6933b.png)

This is referring to the link created by the module to the item. It has an an ARIA role aria-describedby="slick-slide-control01"

The typical use of the ARIA described by attribute is to link a piece of content on a page to the ID of another element on the page  that describes it's function. So if there was a paragraph on the page that described an object, using the aria "described by"  could link those two objects for a screenreader, to avoid redundancy for the user. Like if there is a complex diagram immediately followed by a description of the diagram, or a table immediately followed by an interpretation of the data.
My research on the aria-describedby from Mozilla: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

This is a WCAG 2.1 level A violation, and Deque labels it a "critical" violation.
https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr-value?application=AxeChrome

The use of aria-describedby here seems inappropriate and I suggest it should be removed.

**Comment by beckej13820** [comment]
There's a similar discussion going on in the WordPress Community / WordPress Accessibility Page. I just wanted to share if this issue is under review.

<img width="674" alt="Screen Shot 2022-05-27 at 6 10 41 PM" src="https://user-images.githubusercontent.com/29982981/170795403-c29c8b65-7365-4856-afd7-adb65466b8af.png">

So right now, the aria-describedby isn't doing anything useful because the described-by isn't connecting to an ID on the page.

However, one step forward would be to generate a unique ID for each. So the first link is aria-describedby="slider-title-1" and then the title has the matching id "slider-title-1"
